SharedLibs/TileMapHelper/src/render/PixiHelper.ts

```diff
@@ -174,40 +116,62 @@ export class PixiTileMapHelper {
         const tileHeight = tileLayer.tileMap.getTileHeight();
         const dimensionX = tileLayer.tileMap.getDimensionX();
         const dimensionY = tileLayer.tileMap.getDimensionY();
+        const alpha = tileLayer.getAlpha();
 
         for (let y = 0; y < dimensionY; y++) {
           for (let x = 0; x < dimensionX; x++) {
             const xPos = tileWidth * x;
             const yPos = tileHeight * y;
 
-            const tileId = tileLayer.get(x, y);
-            if (tileId === undefined) {
+            const tileGID = tileLayer.getTileGID(x, y);
+            if (tileGID === undefined) {
               continue;
             }
-            const tileTexture = textureCache.findTileTexture(tileId);
-            if (!tileTexture) {
-              console.warn(`Unknown tile id: ${tileId} at (${x}, ${y})`);
-              continue;
-            }
-            const rotate = textureCache.getPixiRotate(
-              tileLayer.isFlippedHorizontally(x, y),
-              tileLayer.isFlippedVertically(x, y),
-              tileLayer.isFlippedDiagonally(x, y)
-            );
-            const pixiTilemapFrame = pixiTileMap.tile(tileTexture, xPos, yPos, {
-              rotate,
-            });
+            const tileId = FlippingHelper.getTileId(tileGID);
 
             const tileDefinition = tileLayer.tileMap.getTileDefinition(tileId);
 
-            // Animated tiles have a limitation:
-            // they are only able to use frames arranged horizontally one next
-            // to each other on the atlas.
-            if (tileDefinition && tileDefinition.getAnimationLength() > 0) {
-              pixiTilemapFrame.tileAnimX(
-                tileWidth,
-                tileDefinition.getAnimationLength()
+            if (tileDefinition.hasStackedTiles()) {
+              for (const tileGID of tileDefinition.getStackedTiles()) {
+                const tileId = FlippingHelper.getTileId(tileGID);
+                const tileTexture = textureCache.getTexture(tileId);
+                if (!tileTexture) {
+                  continue;
+                }
+
+                const rotate = getPixiRotate(tileGID);
+
+                void pixiTileMap.tile(tileTexture, xPos, yPos, {
```

Comment:
What does `void` means in this context?

Reply:
`void` indicates to typescript that the return value of `pixiTileMap.tile` can be dismissed.

Reply:
done